.env

```diff
@@ -1 +0,0 @@
-PYTHONPATH=apps/api/src
```

Comment:
Question: Is there a reason why you moved the .env? This .env should remain where it is because it's important to the backend.